src/pymatgen/io/abinit/pseudos.py

```diff
@@ -601,7 +601,7 @@ def _dict_from_lines(lines, key_nums, sep=None) -> dict:
     if len(lines) != len(key_nums):
         raise ValueError(f"{lines = }\n{key_nums = }")
 
-    kwargs = Namespace()
```

Comment:
I must have broken `Namespace` dict from `monty` somewhere, also need to double check coverage:

https://github.com/materialsproject/pymatgen/blob/b28b4c24d92830327f2aaf199804b726d680437c/src/pymatgen/io/abinit/pseudos.py#L604-L627